scripts/core/editor3/components/annotations/AnnotationPopup.jsx

```diff
@@ -10,7 +10,9 @@ import {HighlightsPopupPresentation} from '../HighlightsPopupPresentation';
 import {UserAvatar} from 'apps/users/components/UserAvatar';
 
 const Annotation = ({annotation, editorNode, showPopup, highlightId, highlightsManager}) => {
-    const {author, avatar, date, msg, annotationType: type} = annotation.data;
+    const {author, avatar, date, msg, annotationType} = annotation.data;
+    const annotationTypes = ng.get('metadata').values.annotation_types;
```

Comment:
@petrjasek is there any way I can avoid using `ng` to retrieve metadata values?

Reply:
We can add it on store when we init it.

Reply:
So I should use `ng` when we initialize the store then?

Reply:
Yes, I think it is already used there.